docs/_templates/layout.html

```diff
@@ -0,0 +1,17 @@
+{% extends "!layout.html" %}
```

Comment:
Move all the docs files, _templates, conf.py, index.rst to a subdirectory `docs/source/` to be consistent with our other projects documentation and for nepthys to work on this component.